packages/xdl/src/ConnectionStatus.ts

```diff
@@ -0,0 +1,9 @@
+let offline: boolean = false;
+
+export function setIsOffline(bool: boolean): void {
+  offline = bool;
+}
+
+export function isOffline(): boolean {
+  return offline;
+}
```

Comment:
not something important for now but in the future we might want to add a listener to this in case we want to let people switch between online/offline (or listen to network connection and do this automatically) while the cli is running already

Reply:
yeah, i think listening to the network connection and doing it automatically is better. Need to find out the best way to do that where it makes almost no diff in expo-cli performance though